solidity/contracts/bridge/MovingFunds.sol

```diff
@@ -426,4 +428,38 @@ library MovingFunds {
 
         return (targetWalletsHash, outputsTotalValue);
     }
+
+    /// @notice Notifies about a timed out moving funds process. Terminates
+    ///         the wallet and slashes signing group members as a result.
+    /// @param walletPubKeyHash 20-byte public key hash of the wallet
+    /// @dev Requirements:
+    ///      - The wallet must be in the MovingFunds state
+    ///      - The moving funds timeout must be actually exceeded
+    function notifyMovingFundsTimeout(
+        BridgeState.Storage storage self,
+        bytes20 walletPubKeyHash
+    ) external {
+        Wallets.Wallet storage wallet = self.registeredWallets[
+            walletPubKeyHash
+        ];
+
+        require(
+            wallet.state == Wallets.WalletState.MovingFunds,
+            "ECDSA wallet must be in MovingFunds state"
```

Comment:
```suggestion
            "Wallet must be in MovingFunds state"
```

Non-blocking this PR but would be great to align revert messages about wallet states:
- In `Deposit.revealDeposit` we say `Wallet is not in Live state`
- In `Redemption.requestRedemption` we say `Wallet must be in Live state`
- In `Redemption.submitRedemptionProof` we say `Wallet must be in Live or MovingFunds state`
- In `Redemption.notifyRedemptionTimeout` we say `The wallet must be in Live, MovingFunds or Terminated state` <-- something to align as well
- In `Sweep.resolveSweepingWallet` we say `Wallet must be in Live or MovingFunds state`
- In `Fraud.submitFraudChallenge` we say `Wallet is neither in Live nor MovingFunds state`